samples/create_project.py

```diff
@@ -45,23 +45,28 @@ def main():
     logging.basicConfig(level=logging_level)
 
     tableau_auth = TSC.PersonalAccessTokenAuth(args.token_name, args.token_value, site_id=args.site)
-    server = TSC.Server(args.server, use_server_version=True)
+    server = TSC.Server(args.server)
+    server.add_http_options({'verify': False})
+    server.use_server_version()
```

Comment:
why disable verification in the sample?
I don't think we should such an anti-pattern/workaround into a sample which many customers might use as a starting point for their own scripts